pyface/ui/qt4/progress_dialog.py

```diff
@@ -222,6 +222,12 @@ def _create_message(self, dialog, layout):
         self._message_control = label
         return
 
+    def _create_percent(self, dialog, parent_sizer):
+        if not self.show_percent:
+            return
+
+        raise NotImplementedError
```

Comment:
I think I'd be tempted to log or warn and do nothing rather than raise here.

Reply:
I copied the method as it exists in the `wx` backed. Are we going to fix it in both places?

Reply:
If this is the same as Wx, it's fine.